nuxt/plugins/urql.ts

```diff
@@ -126,17 +126,24 @@ export default defineNuxtPlugin(async (nuxtApp) => {
     fetchOptions: () => {
       const store = useMaevsiStore(nuxtApp.$pinia)
       const jwt = store.jwt
+      const turnstileKey = store.turnstileKey
 
+      let headers = {} as Record<string, any>
       if (jwt) {
```

Comment:
```suggestion
      const headers = {} as Record<string, any>
      
      if (jwt) {
```

Reply:
That should also be possible with a `const` object. Then the contents would need to be changed, not the whole object.

Reply:
I changed `let` to `const` (970cbb8612b73372ab6db99eacdd8dd53cc3aba9)